modules/sbo-rbac-requirements.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// *openshift-docs/applications/connecting_applications_to_services/exposing-binding-data-from-a-service
+[id="sbo-rbac-requirements_{context}"]
+= RBAC requirements
+
+To expose the backing service binding data using the {servicebinding-title}, you require certain Role-based access control (RBAC) permissions. You must specify certain verbs under the `rules` field of the `ClusterRole` resource to allow the RBAC persmissions for the backing service resources. Defining these `rules` enables the {servicebinding-title} with permissions to read the binding data of the the backing service resources throughout the cluster.
+
+The {servicebinding-title} performs requests against Kubernetes API using a dedicated service account. By default this account has permissions to bind services to workloads, both represented by the following standard Kubernetes objects:
```

Comment:
```suggestion
The {servicebinding-title} performs requests against the(?) Kubernetes API using a dedicated service account. By default, this account has permissions to bind services to workloads, both represented by the following standard Kubernetes objects:
```
If articles aren't used with APIs throughout this set of docs, then ignore my suggestion, please.